pkg/health/client/modules.go

```diff
@@ -60,3 +69,52 @@ func GetAndFormatModulesHealth(w io.Writer, clt ModulesHealth, verbose bool) {
 		tally[cell.StatusUnknown],
 	)
 }
+
+func buildTree(n *node, raw string) error {
+	var sn cell.StatusNode
+	if err := json.Unmarshal([]byte(raw), &sn); err != nil {
+		return err
+	}
+	build(n, &sn)
+	return nil
+}
+
+func ensurePath(n *node, pp []string) *node {
+	current := n
+	for i := 0; i < len(pp); i++ {
+		if v := current.find(pp[i]); v != nil {
+			current = v
+			continue
+		}
+		name := strings.Replace(pp[i], "(/)", "", 1)
+		current = current.addBranch(name)
+	}
+
+	return current
+}
+
+func build(n *node, sn *cell.StatusNode) {
```

Comment:
With this, I think the only use of `cell.(*UpdateNode).String()` family of functions is for a debug log in hive/health.go.

We should probably just get rid of that code to clean things up (doesn't have to be part of this PR - but lets make it a point to circle back and clean that up).

Reply:
Right I was wondering about that too. Don't think this is within the scope of this PR tho
Additionally tho the output has been enhance our messaging is lame. ie OK OK or worse no messages!. This need TLC and I think this should be our next priority!

Reply:
Yeah right now the modular health updates are making debug logs way too noisy.  With that one we should just update the subreporter node that changed instead of formatting out the whole tree.